src/components/basic/text/Text.stories.tsx

```diff
@@ -33,14 +33,25 @@ const Common: Partial<Story> = {
 	}
 };
 
-export const Size: Story = {
+export const Base: Story = {
```

Comment:
Honestly, I would leave the Base story with all the defaults, and add only the required args (the children in this case)

Reply:
The Base could be the one extended by the others then

Reply:
@bea I directly moved the argTypes in the meta object and renamed the Common story in Base. I think it's clearer in this way. What do you think?

Reply:
Much cleaner :rocket: